config/config.go

```diff
@@ -246,3 +246,12 @@ func GetEtcdServerAddrs() []string {
 	}
 	return []string{"http://localhost:2379"}
 }
+
+func IsTLSenabled() bool {
```

Comment:
I mean if the configuration file have TLS X509 certificate so we use it
If you don't want .... basically we don't need an extra flag/config